src/transformers/models/llava_next/image_processing_llava_next.py

```diff
@@ -446,6 +446,35 @@ def get_image_patches(
 
         return image_patches
 
+    def pad(
+        self,
+        pixel_values: List[np.ndarray],
+        image_sizes: List[Tuple[int, int]],
+        return_tensors: Optional[Union[str, TensorType]] = None,
+    ):
+        """
+        Args:
+            pixel_values (`List[np.ndarray]`): array of pixel values of each images
+            image_sizes (`List[Tuple[int, int]]`): image sizes (height, weidt)
+            return_tensors (`str` or `TensorType`, *optional*):
+                The type of tensors to return. Can be one of:
+                - Unset: Return a list of `np.ndarray`.
+                - `TensorType.TENSORFLOW` or `'tf'`: Return a batch of type `tf.Tensor`.
+                - `TensorType.PYTORCH` or `'pt'`: Return a batch of type `torch.Tensor`.
+                - `TensorType.NUMPY` or `'np'`: Return a batch of type `np.ndarray`.
+                - `TensorType.JAX` or `'jax'`: Return a batch of type `jax.numpy.ndarray`.
+        """
+        max_patch = max(len(x) for x in pixel_values)
+        pixel_values = [
+            np.concatenate([x, np.zeros([max_patch - x.shape[0]] + list(x.shape[1:]), dtype=x.dtype)], axis=0)
+            if x.shape[0] < max_patch
+            else x
+            for x in pixel_values
+        ]
+        data = {"pixel_values": pixel_values, "image_sizes": image_sizes}
```

Comment:
Do you mean if `image_sizes` is not None, because it cannot be None and is obtained at the very beginning, when we try to make patches for single images